web/server.py

```diff
@@ -7,6 +7,46 @@
 
 app = Flask(__name__, static_url_path='/static')
 
+@app.route('/api/getdata', methods=['POST'])
+def getdata():
+     username = request.form['username']
+     conn = sqlite3.connect('/home/ubuntu/database.db')
+     c = conn.cursor()
+
+     out = c.execute("SELECT * FROM data where usr=?", (username))
+
+     if(out.fetchone() == None):
+          conn.close()
+          return json.dumps({"class": "Knight", "items": "{}", "health": 100.0, "stamina": 100.0, "mana": 100.0, "posx": 0.0, "posy": 0.0}), 200
```

Comment:
if none, then return 666?

Reply:
No, because the player has no data, so it just sends back some default values